exportdb/permissions.py

```diff
@@ -1,10 +1,11 @@
 import rules
-# TODO: this should be parametrized via settings
-
+from django.conf import settings
 
 @rules.predicate
-def is_superuser(user):
-    return user.is_superuser
+def permission(*args, **kwargs):
+    predicate = getattr(settings, 'EXPORTDB_PERMISSION', rules.is_superuser)
```

Comment:
I think we should rather do something like:

```
# in exportdb/conf.py
class ExportDBConf(AppConf):
    EXPORT_ROOT = os.path.join(settings.MEDIA_ROOT, EXPORT_SUBDIR)
    EXPORT_MEDIA_URL = posixpath.join(settings.MEDIA_URL, EXPORT_SUBDIR)
    EXPORT_CONF = {}
    PERMISSION = 'rules.is_superuser'  # add the default

    # form used in admin view to confirm export
    CONFIRM_FORM = 'django.forms.Form'
```

with
```
# in exportdb/permissions.py
...
from django.conf import settings

import rules

from .compat import import_string


@rules.predicate
def permission(*args, **kwargs):
    predicate = import_string(settings.EXPORTDB_PERMISSION)
    return predicate(*args, **kwargs)


rules.add_rule('exportdb.can_export', permission)
```

This should cover the desired flexibility for now and I'll have look at cleaner permission handling later. Rules is a very powerful library where you can logically combine predicates, we should definitely leverage that.

On a sidenote, please run pylint and PEP8 on the code :)